tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -462,6 +480,8 @@ void handleRequestImpl(sourcekitd_object_t ReqObj, ResponseReceiver Rec) {
     Opts.EnableStructure = EnableStructure;
     Opts.EnableDiagnostics = EnableDiagnostics;
     Opts.SyntaxTransferMode = syntaxTransferModeFromUID(TransferModeUID);
+    Opts.SyntaxSerializationFormat =
+        syntaxSerializationFormatFromUID(SerializationFormatUID);
```

Comment:
This and `syntaxTransferModeFromUID` should return `Optional`, and return error response for invalid UID.